### PR TITLE
add cookbook for mkl multiclass

### PR DESCRIPTION
**Pull request by sanuj**
Will add integration data once code is reviewed.

### CODE REVIEW

File: doc/cookbook/source/examples/classifier/multiple_kernel_learning_multiclass.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
+======================================================
+Multiple Kernel Learning for Multi-class Classification
```
Comment (karlnapf) [inline doc/cookbook/source/examples/classifier/multiple_kernel_learning_multiclass.rst]: remove "for multiclass classification"

File: doc/cookbook/source/examples/classifier/multiple_kernel_learning_multiclass.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+======================================================
+
+Multiple kernel learning (MKL) is based on convex combinations of arbitrary kernels over potentially different domains.
+
+.. math::
+
+    {\bf k}(x_i,x_j)=\sum_{i=1}^{K} \beta_k {\bf k}_i(x_i, x_j)
+
+where :math:`\beta_k > 0`, :math:`\sum_{k=1}^{K} \beta_k = 1`, :math:`K` is the number of sub-kernels, :math:`\bf{k}` is a combined kernel, :math:`{\bf k}_i` is an individual kernel and :math:`{x_i}_i` are the training data.
```
Comment (karlnapf) [inline doc/cookbook/source/examples/classifier/multiple_kernel_learning_multiclass.rst]: Same as for the other cookbook, reference to binary case would be good

File: doc/cookbook/source/examples/classifier/multiple_kernel_learning_multiclass.rst [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+Classification is done by using :sgclass:`CMulticlassSVM`.
```
Comment (karlnapf) [inline doc/cookbook/source/examples/classifier/multiple_kernel_learning_multiclass.rst]: Basically all comments for  the other cookbook are also valid here

File: doc/cookbook/source/examples/classifier/multiple_kernel_learning_multiclass.rst (hunk 2 above)
Comment (sanuj) [inline doc/cookbook/source/examples/classifier/multiple_kernel_learning_multiclass.rst]: binary mkl cookbook?

Comment (karlnapf) [inline doc/cookbook/source/examples/classifier/multiple_kernel_learning_multiclass.rst]: maybe yeah

File: doc/cookbook/source/examples/classifier/multiple_kernel_learning_multiclass.rst (hunk 3 above)
Comment (sanuj) [inline doc/cookbook/source/examples/classifier/multiple_kernel_learning_multiclass.rst]: I would like to reference multiclass svm cookbook here. #3314

Comment (karlnapf) [inline doc/cookbook/source/examples/classifier/multiple_kernel_learning_multiclass.rst]: do it

File: examples/meta/src/classifier/multiple_kernel_learning_multiclass.sg [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+#![create_combined_train]
+CombinedKernel combined_kernel()
+combined_kernel.append_kernel(poly_kernel)
+combined_kernel.append_kernel(gauss_kernel)
+combined_kernel.append_kernel(linear_kernel)
+combined_kernel.init(features_train, features_train)
+#![create_combined_train]
+
+#![train_mkl]
+real C = 1
```
Comment (sorig) [inline examples/meta/src/classifier/multiple_kernel_learning_multiclass.sg]: `1.0` please :)

**Comment by karlnapf** [comment]
Hi @sanuj 
Can we get this merged soon?

**Comment by sanuj** [comment]
@karlnapf Will do it till this weekend if that's fine. :slightly_smiling_face:

**Comment by karlnapf** [comment]
Would be cool to get this one in the end!

**Pull request closed without merging**